src/main/java/seedu/weme/model/meme/Description.java

```diff
@@ -20,20 +20,20 @@
     public final String value;
 
     /**
-     * Constructs an {@code Address}.
+     * Constructs a {@code Description}.
      *
-     * @param address A valid weme.
+     * @param description A valid weme.
      */
-    public Address(String address) {
-        requireNonNull(address);
-        checkArgument(isValidAddress(address), MESSAGE_CONSTRAINTS);
-        value = address;
+    public Description(String description) {
+        requireNonNull(description);
+        checkArgument(isValidDescription(description), MESSAGE_CONSTRAINTS);
+        value = description;
     }
 
     /**
-     * Returns true if a given string is a valid address.
+     * Returns true if a given string is a valid description.
      */
-    public static boolean isValidAddress(String test) {
+    public static boolean isValidDescription(String test) {
```

Comment:
Actually I think any string is a valid description. With the empty string as an indication that there was no input. Thoughts??

Reply:
If user wants description to be empty then shouldn't provide the prefix. Error only occurs when user provides description prefix but gives empty input

Reply:
hm.. then what if they wanted to set the description to be an empty string, like they wanted to clear the description via an edit command, this would prevent it right?

Reply:
Nope, working on the regex now, and passing the test cases